ui/src/components/WelcomeTutorial.tsx

```diff
@@ -425,7 +425,6 @@ export function WelcomeTutorial({
         onClose={handleClose}
       >
         <StyledFilterArrowTip src={DialogTip}></StyledFilterArrowTip>
-        <StyledFilterOverlay src={BigOverlayCircle}></StyledFilterOverlay>
```

Comment:
Can you remove the import for 'BigOverlayCircle' and declaration for 'StyledFilterOverlay' since you aren't using them anymore?

Reply:
oop i have the same thing as megan